__tests__/common/formatHelpers/sortByReference.test.js

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with
+ * the License. A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+import { expect } from 'chai';
+import sortByReference from '../../../lib/common/formatHelpers/sortByReference.js';
+
+const DEFAULT_TRANSFORMED_TOKENS = {
+  color: {
+    primary: {
+      value: '#FF0000',
+      type: 'color',
+      original: { value: '{color.red}', type: 'color' },
+    },
+    red: {
+      value: '#FF0000',
+      type: 'color',
+      original: { value: '#FF0000', type: 'color' },
+    },
+  },
+};
+
+const DTCG_TRANSFORMED_TOKENS = {
+  color: {
+    primary: {
+      $value: '#FF0000',
+      $type: 'color',
+      original: { $value: '{color.red}', $type: 'color' },
+    },
+    red: {
+      $value: '#FF0000',
+      $type: 'color',
+      original: { $value: '#FF0000', $type: 'color' },
+    },
+  },
+};
```

Comment:
```suggestion
const TRANSFORMED_TOKENS = (dtcg) => ({
  color: {
    primary: {
      [`${dtcg ? '$' : ''}value`]: '#FF0000',
      [`${dtcg ? '$' : ''}type`]: 'color',
      original: { 
        [`${dtcg ? '$' : ''}value`]: '{color.red}',
        [`${dtcg ? '$' : ''}type`]: 'color' },
    },
    red: {
      [`${dtcg ? '$' : ''}value`]: '#FF0000',
      [`${dtcg ? '$' : ''}type`]: 'color',
      original: { 
        [`${dtcg ? '$' : ''}value`]: '#FF0000',
        [`${dtcg ? '$' : ''}type`]: 'color' },
    },
  },
});
```

nitpick suggestion, could simplify this, up to you

Reply:
Sure, looks fine 👍